src/main/java/io/jenkins/plugins/analysis/core/steps/RecordIssuesStep.java

```diff
@@ -68,6 +68,7 @@
     private boolean ignoreQualityGate = false; // by default, a successful quality gate is mandatory;
     private boolean ignoreFailedBuilds = true; // by default, failed builds are ignored;
     private String referenceJobName;
+    private String referenceBuildId;
```

Comment:
Initialize with ` = StringUtils.EMPTY` (and add the same thing for line 70 since otherwise the snipped generator will generate an empty statement even if the string is empty.

Reply:
Done